geode-core/src/main/java/org/apache/geode/internal/cache/wan/AbstractGatewaySenderEventProcessor.java

```diff
@@ -862,14 +868,14 @@ private void logEvents(String message, List<GatewaySenderEventImpl> events) {
               event.getRawCallbackArgument(), this.sender.getMyDSId(), allRemoteDSIds);
           event.setCallbackArgument(geCallbackArg);
           GatewaySenderEventImpl pdxSenderEvent =
-              new GatewaySenderEventImpl(EnumListenerEvent.AFTER_UPDATE, event, null); // OFFHEAP:
-                                                                                       // event for
-                                                                                       // pdx type
-                                                                                       // meta data
-                                                                                       // so it
-                                                                                       // should
-                                                                                       // never be
-                                                                                       // off-heap
+              new GatewaySenderEventImpl(EnumListenerEvent.AFTER_UPDATE, event, null, false); // OFFHEAP:
+          // event for
+          // pdx type
+          // meta data
+          // so it
+          // should
+          // never be
+          // off-heap
```

Comment:
I have put it in a single line above the instance creation sentence. I cannot tell if the comment is significant so I do not know if it can be removed.